Rakefile

```diff
@@ -40,6 +45,14 @@ task :console => :print_env do
   sh 'pry -r ./specs/test_load_all -e \'cd CoEditPDF\''
 end
 
+namespace :newkey do
```

Comment:
Should also call `SecureDB.generate_salt` in this task (or create another task) for generating `DB_SALT`.